.vscode/launch.json

```diff
@@ -1,6 +1,20 @@
 {
```

Comment:
You should not check in files from .vscode folder because those may be your specific configurations and you will overwrite other people's configurations.